### PR TITLE
DATAUP-330: eslint, formatting, and variable name standardisation

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of PR purpose/changes

Prior to looking at test failures in the `truss` branch, I have applied standardised formatting to component files and formatting and variable naming to the test files.

- Ran eslint and prettier on files in `/static/kbase/js/common/cellComponents/` and `test/unit/spec/common/cellComponents/`
- In test files, named the objects produced by each module `<module_name>Instance`

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-X
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [ ] Tests pass locally and in GitHub Actions -- tests are failing at present!
- [ ] Changes available by spinning up a local narrative and navigating to _X_ to see _Y_

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook

### CODE REVIEW

File: test/unit/spec/common/cellComponents/fieldTableCellWidget-Spec.js [hunk 1]
```diff
@@ -5,21 +5,21 @@ define([
     'common/props',
     'common/spec',
     '/test/data/testAppObj',
-], (Jupyter, fieldCellWidget, ParamResolver, Props, Spec, TestAppObject) => {
+], (Jupyter, FieldCellWidget, ParamResolver, Props, Spec, TestAppObject) => {
```
Comment (ialarmedalien) [inline test/unit/spec/common/cellComponents/fieldTableCellWidget-Spec.js:8]: Standardisation: use title case for module names

File: test/unit/spec/common/cellComponents/fieldTableCellWidget-Spec.js [hunk 2]
```diff
@@ -5,21 +5,21 @@ define([
...
 
         it('has expected functions', () => {
-            expect(fieldCellWidget.make).toBeDefined();
+            expect(FieldCellWidget.make).toBeDefined();
         });
     });
 
     describe('The Field Table Cell Widget instance', () => {
-        let node, mockFieldWidget;
+        let node, fieldCellWidgetInstance;
```
Comment (ialarmedalien) [inline test/unit/spec/common/cellComponents/fieldTableCellWidget-Spec.js:22]: standardisation: name the objects produced by a module `${moduleName.toCamelCase()}.Instance`

File: test/unit/spec/common/cellComponents/fieldTableCellWidget-Spec.js [hunk 3]
```diff
@@ -127,21 +127,22 @@ define([
...
         });
 
         it('has the required methods', () => {
-            expect(mockFieldWidget.start).toBeDefined();
-            expect(mockFieldWidget.stop).toBeDefined();
-            expect(mockFieldWidget.bus).toBeDefined();
+            ['bus', 'start', 'stop'].forEach((fn) => {
+                expect(fieldCellWidgetInstance[fn]).toBeDefined();
+                expect(fieldCellWidgetInstance[fn]).toEqual(jasmine.any(Function));
+            });
```
Comment (ialarmedalien) [inline test/unit/spec/common/cellComponents/fieldTableCellWidget-Spec.js:141]: minor refactor

File: test/unit/spec/common/cellComponents/filePathWidget-Spec.js [hunk 4]
```diff
@@ -59,22 +59,24 @@ define([
...
         });
 
         it('has the required methods', () => {
-            expect(filePathWidget.start).toBeDefined();
-            expect(filePathWidget.stop).toBeDefined();
-            expect(filePathWidget.bus).toBeDefined();
+            ['bus', 'start', 'stop'].forEach((fn) => {
+                expect(filePathWidgetInstance[fn]).toBeDefined();
+                expect(filePathWidgetInstance[fn]).toEqual(jasmine.any(Function));
+            });
```
Comment (ialarmedalien) [inline test/unit/spec/common/cellComponents/filePathWidget-Spec.js:75]: refactor

File: test/unit/spec/common/cellComponents/filePathWidget-Spec.js [hunk 5]
```diff
@@ -30,7 +30,7 @@ define([
             Jupyter.narrative = null;
         });
 
-        let filePathWidget, node, spec, parameters;
+        let filePathWidgetInstance, node, spec, parameters;
```
Comment (ialarmedalien) [inline test/unit/spec/common/cellComponents/filePathWidget-Spec.js:33]: name standardisation

File: test/unit/spec/common/cellComponents/tabs/infoTab-Spec.js [hunk 6]
```diff
@@ -1,13 +1,13 @@
-define(['common/cellComponents/tabs/infoTab'], (infoTabWidget) => {
+define(['common/cellComponents/tabs/infoTab'], (InfoTab) => {
```
Comment (ialarmedalien) [inline test/unit/spec/common/cellComponents/tabs/infoTab-Spec.js:1]: module name standardisation

File: test/unit/spec/common/cellComponents/tabs/infoTab-Spec.js [hunk 7]
```diff
@@ -43,31 +43,33 @@ define(['common/cellComponents/tabs/infoTab'], (infoTabWidget) => {
             },
         };
         const appSpec = model.getItem('app.spec');
-        const mockInfoTab = infoTabWidget.make({ model });
+        const infoTabInstance = InfoTab.make({ model });
```
Comment (ialarmedalien) [inline test/unit/spec/common/cellComponents/tabs/infoTab-Spec.js:46]: instance name standardisation

**Review by briehl — APPROVED** [review]
Some good formatting and standardization!

**Pull request merged**